lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -382,6 +382,32 @@ def cmd_hosts(*args)
             return
           end
         }
+
+      when '-g'
+        # This product includes GeoLite data created by MaxMind, available from http://www.maxmind.com
+        host = args.shift
+        # RegEx#match? seems to be faster on newer versions of Ruby
+        if !Rex::Socket::MATCH_IPV4.match?(host) && !Rex::Socket::MATCH_IPV6.match?(host)
+          if Rex::Socket::MATCH_IPV4_PRIVATE.match?(host)
+            print_error("You cannot geolocate local IP addresses")
+            return
+          end
+          print_error("#{host} is not a valid IP address")
+          return
+        end
+        glcdat = "#{File.expand_path(File.dirname(__FILE__))}/../../../../../data/GeoLiteCity.dat"  # Point to data directory
```

Comment:
@thecarterb Sorry for the mistake. The file name should be ```GeoLiteCity.dat``` and not ```GeoCity.dat```. Please correct the file name in the updated commit.